Code/BuildSystem/CMake/CMakeUtils/ezUtilsPlatformVars.cmake

```diff
@@ -0,0 +1,11 @@
+# Cmake variables which are platform dependent
+
+# #####################################
+# ## General settings
+# #####################################
+if(EZ_CMAKE_PLATFORM_WINDOWS OR EZ_CMAKE_PLATFORM_LINUX)
```

Comment:
Why was this moved into a separate file?

Reply:
Because otherwise EZ_CMAKE_PLATFORM_WINDOWS and EZ_CMAKE_PLATFORM_LINUX are not defined yet.